srcpkgs/Solaar/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'Solaar'
 pkgname=Solaar
-version=1.1.5
-revision=2
+version=1.1.7
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-evdev python3-pyudev python3-psutil python3-yaml
```

Comment:
gtk+3 is missing

Reply:
Right, this is something I wasn't sure about when I last updated it as well. I guess the maintainer didn't included it initially cause the cli works fine without it? I don't have a preference either way.

Reply:
I don't use this software, so I assumed it's only graphical software.

Reply:
Could we split CLI to a subpackage?

Reply:
The cli doesn't seem to be something that can be easily separated. Like if you just run `$ solaar` it will open a gui. But the functionalities can be achieved with `$ solaar show`, `$ solaar pair`, `$ solaar probe` etc subcommands.